src/Symfony/Component/Dotenv/Tests/Command/Fixtures/Scenario4/composer.json

```
@@ -0,0 +1,7 @@
+{
+    "extra": {
+        "runtime": {
+            "dotenv_path": "sub/path/.env"
```

By using a different file name, the test will cover more cases.
```suggestion
            "dotenv_path": "sub/path/custom.env"
```